--- origsrc/gnome-mastermind-0.3.1/Makefile.am	2007-05-23 11:19:20.000000000 -0500
+++ src/gnome-mastermind-0.3.1/Makefile.am	2008-04-28 22:39:29.875000000 -0500
@@ -25,7 +25,7 @@
 
 DISTCHECK_CONFIGURE_FLAGS = --disable-scrollkeeper
 
-pkgdata_DATA =  AUTHORS    \
+noinst_DATA =  AUTHORS    \
 	   	BUGS       \
 	COPYING    \
 	copyright  \
@@ -33,5 +33,3 @@
 	NEWS       \
 	README     \
 	TODO       
-
-ACLOCAL_AMFLAGS = -I m4
